test/setup/versions.tf

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
Do not change this pinning approach. It is how we should do pinning within `test/setup`.